### PR TITLE
Improving the description of `const` usage

### PR DESCRIPTION
**Pull request by justsml**
Using CAPITAL_CASE with `const` should be rare (one valid use-case is Redux's named events.)

The confusion around `const` & the UPPER_CASE naming pattern comes from other languages - C, Java, etc.

### CODE REVIEW

**Comment by justsml** [comment]
@metagrover 
Let me know if you want me to find the TC39 discussion on this topic. I think it was in the meeting notes somewhere...

**Comment by justsml** [comment]
To clarify, I remove the word `frozen` because it assumes deeper understanding of JS.

I tried to emphasize that re-assigning the variable is what's dis-allowed.

File: README.md [hunk 1]
```diff
@@ -54,27 +54,27 @@ var a = 2;
...
+    console.log(arr); // [3,6,7]
 }
 ```
 
 A few things to keep in mind:
 
 * Hoisting of `let` and `const` vary from the traditional hoisting of variables and functions. Both `let` and `const` are hoisted, but cannot be accessed before their declaration, because of [Temporal Dead Zone](http://jsrocks.org/2015/01/temporal-dead-zone-tdz-demystified)
 * `let` and `const` are scoped to the nearest enclosing block.
-* When using `const`, use CAPITAL_CASING (this is a common convention).
+* When using `const` with fixed strings, CAPITAL_CASING might be appropriate.
```
Comment (metagrover) [inline README.md]: I agree with you on this. I think a better explanation would be:

When using `const` with fixed strings or values, CAPITAL_CASING might be appropriate (ex: `const PI = 3.14`)

Still not sure, if "fixed" is doing justice to the statement here though 🤔

**Review by metagrover — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (justsml) [reply README.md]: Good point, I'll adjust...

**Comment by metagrover** [comment]
At the time of writing this guide, I remember having this discussion with someone here or on reddit about using const variables with CAPITAL_CASING. I can see that MDN Docs also list this as a common convention in an [example](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const). But, I completely agree with your thoughts on this.

**Comment by justsml** [comment]
MDN is on point for virtually all their documentation.

However, `const` & `let` is still a frustratingly misguided article. Last I checked it still seems like it has something to do with hoisting - just not the case.

**Review by metagrover — APPROVED** [review]

**Comment by justsml** [comment]
### Haha, this bit here makes me think a Java/C developer is leaking "their conventions" into MDN docs:

image

**Comment by justsml** [comment]
Thanks again for the merge @metagrover !!! 

Much appreciated!

**Pull request merged**